pkg/flags/flags_test.go

```diff
@@ -109,3 +109,37 @@ func TestMaxmindRetryDownload(t *testing.T) {
 		t.Fatalf("Expected an error parsing flags but none returned")
 	}
 }
+
+func TestDisableLeaderElectionFlag(t *testing.T) {
+	ResetForTesting(func() { t.Fatal("Parsing failed") })
+
+	oldArgs := os.Args
+	defer func() { os.Args = oldArgs }()
+	os.Args = []string{"cmd", "--disable-leader-election", "--http-port", "80", "--https-port", "443"}
+
+	_, conf, err := ParseFlags()
+	if err != nil {
+		t.Fatalf("Unexpected error parsing default flags: %v", err)
+	}
+
+	if !conf.DisableLeaderElection {
+		t.Fatalf("Expected --disable-leader-election and conf.DisableLeaderElection as true, but found: %v", conf.DisableLeaderElection)
+	}
+}
+
+func TestDisableLeaderElectionDisabledByDefault(t *testing.T) {
```

Comment:
this means that the flag is set to false correct? this test name is a bit confusing

Reply:
I'm rename this test to `TestIfLeaderElectionHasDisabledByDefault`, what do you think?

Reply:
`TestIfLeaderElectionDisabledFlagIsFalse` i guess, naming is hard

Reply:
I don't know if `TestIfLeaderElectionDisabledFlagIsFalse` is a good idea, because the test intent is checking the default value when the flag is not present on cmd. In this case the `...ByDefault` makes more sense.?

Reply:
sgtm

Reply:
thanks!